### PR TITLE
Send fastft errors to sentry, and cache status to graphite

### PR DESCRIPTION
**Pull request by adgad**

### CODE REVIEW

File: server/lib/cache.js [hunk 1]
```diff
@@ -32,9 +34,12 @@ class Cache {
...
 
 		// we have fresh data
-		if(expire > now && data) { return Promise.resolve(data); }
-
+		if(expire > now && data) {
+			metrics.count(`cacher.${key}.cached`, 1);	
+			return Promise.resolve(data);
+		}
 		// we don't have fresh data, fetch it
+		metrics.count(`cacher.${key}.fresh`, 1);
```
Comment (charypar) [inline server/lib/cache.js]: This is not an entirely accurate picture, most of the time we return stale below. May be worth counting separately?

**Comment by wheresrhys** [comment]
I just released a version of express-errors-handler that's a bit more intelligent about choosing to send to splunk/sentry. Might be worth using the new stuff from there, or adding another imilar method for use outside of an express controller

**Comment by adgad** [comment]
Ah cool, I can do that independently of this (i.e. wrap captureError in express-errors-handler). Will do that now.

**Comment by adgad** [comment]
@charypar This okay to merge now?

File: package.json [hunk 2]
```diff
@@ -20,6 +21,7 @@
     "next-ft-api-client": "^3.10.1"
   },
   "devDependencies": {
+		"bower": "~1.4.1",
```
Comment (charypar) [inline package.json]: weird whitespace?

**Comment by charypar** [comment]
:+1:

**Pull request merged**